modules/imgproc/src/contours_new.cpp

```diff
@@ -271,59 +277,60 @@ static void icvFetchContourEx(Mat &image,
 struct ContourScanner_
 {
     Mat image;
-    Point offset;             /* ROI offset: coordinates, added to each contour point */
-    Point pt;                 /* current scanner position */
-    Point lnbd;               /* position of the last met contour */
-    schar nbd;                    /* current mark val */
-    int approx_method1;         /* approx method when tracing */
-    int approx_method2;         /* final approx method */
+    Point offset; /* ROI offset: coordinates, added to each contour point */
+    Point pt; /* current scanner position */
+    Point lnbd; /* position of the last met contour */
+    schar nbd; /* current mark val */
+    int approx_method1; /* approx method when tracing */
+    int approx_method2; /* final approx method */
```

Comment:
`do not touch` is preferable here too.

2 or more spaces before comment should help with reading code.

Reply:
This would require setting option `SpacesBeforeTrailingComments: 2` **and** changing comments to `//`- I'll apply these changes.